pkg/controller/constrainttemplate/constrainttemplate_controller_test.go

```diff
@@ -540,18 +540,14 @@ func TestReconcile(t *testing.T) {
 			logger.Error(err, "create cstr")
 			t.Fatal(err)
 		}
-		err = retry.OnError(testutils.ConstantRetry, func(_ error) bool {
```

Comment:
dont we still need the retry since creating vapbinding can take some time.

Reply:
Added the retries.